### ISSUE TITLE
Add "Allowed connections" tab

### ISSUE THREAD

**Issue by anwar-alsilwy** [tnodir/fort#27]
Hi
What do you think about adding "Allowed connections" tab to Statistics UI, and make it similar to "Blocked connections" window, with same details like (IP address, direction, host name and Auto scroll,... etc).
It will be so useful for network diagnostics and make FortFirewall more professional.

![20220516_052712](https://user-images.githubusercontent.com/71539261/168511009-6952c8ba-1f1d-46ad-b1f6-6e260c0e0153.jpg)

**Comment by tnodir** [comment]
Yes, it’s [planned](https://github.com/tnodir/fort/projects/1#card-55583462) in the [TODO list](https://github.com/tnodir/fort/projects/1).

**Comment by anwar-alsilwy** [comment]
Ok, you do your best,
Thanks for your time.
